Node/core/lib/botbuilder.d.ts

```diff
@@ -47,6 +47,11 @@ export type MatchType = RegExp | string | (RegExp | string)[];
  */
 export type ValueListType = string | string[];
 
```

Comment:
Adding semanticActionStates literal types to the botbuilder types definition file. Considered doing this via enums, however defining the values would be considered 'implementation'.